tasks/init-setup.yml

```diff
@@ -4,6 +4,20 @@
     path: "{{ gogs_user_home }}/{{ gogs_init_script_path }}"
     mode: 0755
 
+- name: Set path to Gogs in init script.
+  lineinfile:
+    dest: "{{ gogs_user_home }}/{{ gogs_init_script_path }}"
+    line: "WORKINGDIR={{ gogs_user_home }}/gogs"
+    regexp: "^WORKINGDIR=.*"
```

Comment:
I'd rather combine these into one task, with `with_items`. See this example: https://github.com/geerlingguy/ansible-role-apache/blob/master/tasks/configure-RedHat.yml#L2-L9